instruqt/delivering-as-an-appliance/02-customize-for-your-app/assignment.md

```diff
@@ -16,6 +20,184 @@ timelimit: 300
+kind: Application
+metadata:
+  name: slackernews
+spec:
+  title: SlackerNews
+  icon: "https://uploads-ssl.webflow.com/6310ad0e6a18aa1620da6ae8/6330e04f42bc6a7ba03b4725_snicon.png"
+```
+
+There are other things we can configure in this file as well, but for this lab
+we'll focus just on the title and the icon. For other possible properties, take
+a look at the [reference
+documentation](https://docs.replicated.com/reference/custom-resource-application).
+Not all of the possible values apply for the Kubernetes appliance experience.
+
+The appliance installation process also supports another resource that
+describes application metadata. The resource is based defined by the
+Kubernetes Apps Special Interest Group (SIG) and is also called `Application`.
+This `Application` resource is not unique to the Replicated Platform. It's
+also not required for the appliance installation experience.
+
+We'll start with adding the basic Replicated `Application` resource for basic
+branding, then we'll expand on it a bit. We'll create a SIG Application
+resource as part of that expansion.
+
+### Add the Application Manifest
+
+Let's go back to the "Release Editor" tab and add a new file for the
+application metadata. Create a new file in the `release` directory named
+`replicated-app.yaml` and copy/paste the contents from the example above into
+it.
+
+![Adding the application metadata manifest](../assets/adding-application-manifest.png)
+
+This will add the name and logo to the appliance installation experience and
+the Admin Console.
+
+Enriching the Admin Console
+===========================
+
+To improve the Slackernews appliance experience, we'll configure an
+application status indicator. This will help an administrator understand the
+running state of the application and quickly connect to it.
+
+### Reporting Status on the Admin Console
+
+Application status is reported based on status informers identified in the
+Replicated `Application` resource. Status informers help the Admin Console
+determine the application status to display. The status is an aggregation
+across the provided status informers.
+
+Application status is similar to the status you see for a Kubernetes
+resource when using the `kubectl` command. The statuses provided are based on
+the resources configured as status informers.
+
+<table>
+<thead>
+</tr><th>Status</th><th>Description</th></tr>
+</thead>
+<tbody>
+<tr><td>Ready</td><td>All of the resources are ready</td></tr>
+<tr><td>Unvailable</td><td>One or more of the resources are unavailable</td></tr>
+<tr><td>Degraded</td><td>One or more of the resources are in a degraded state</td></tr>
+<tr><td>Updating</td><td>There are multiple versions of the some resource present</td></tr>
+<tr><td>Missing</td><td>The resources have not yet been created</td></tr>
+</tbody>
+</table>
+
+Status informers can reference on of several types of resource:
+
+* Deployments
+* Stateful sets
+* Daemon sets
+* Services
+* Persistent volume claims
```

Comment:
```suggestion
* StatefulSets
* DaemonSets
* Services
* PersistentVolumeClaims
```

Reply:
it's interesting. I deliberately didn't use the resource syntax and chose English. I'm curious why you opted the other way.

Reply:
Eavesdropping: I like using the resource names in the docs so there's no confusion about the types of resources supported (most likely people know that Persistent volume claim = PersistentVolumeClaim, but using PersistentVolumeClaim slightly reduces the level of thinking required)